docs/topics/request-response.rst

```diff
@@ -115,21 +115,18 @@ Request objects
         cookies for that domain and will be sent again in future requests.
         That's the typical behaviour of any regular web browser.
 
-        To create a request that does not send stored cookies and does not
-        store received cookies, set the ``dont_merge_cookies`` key to ``True``
-        in :attr:`request.meta <scrapy.Request.meta>`.
-
-        Example of a request that sends manually-defined cookies and ignores
-        cookie storage:
+        Example of a request that sends manually-defined cookies:
 
         .. code-block:: python
 
             Request(
                 url="http://www.example.com",
                 cookies={"currency": "USD", "country": "UY"},
-                meta={"dont_merge_cookies": True},
             )
```

Comment:
The only purpose of this example here was to feature dont_merge_cookies, since there is already an example of cookie use higher up. So I think you can remote this example (and the introductory line) altogether.

Reply:
Sure, I'll remove it.